lms/static/sass/discussion/utilities/_v1-compatibility.scss

```diff
@@ -0,0 +1,198 @@
+// Utilities to provide v1-styling compatibility
+
+@mixin font-size($sizeValue: 16){
+  font-size: $sizeValue + px;
+}
+
+@mixin line-height($fontSize: auto){
+  line-height: ($fontSize*1.48) + px;
+}
+
+%text-sr {
+    // clip has been deprecated but is still supported
```

Comment:
Should probably test this in Edge, browsers that built their engines before this was deprecated will probably have it forever but who knows about ones that were built more recently...